go-controller/pkg/ovn/loadbalancer.go

```diff
@@ -188,14 +188,28 @@ func (ovn *Controller) createLoadBalancerRejectACL(lb string, sourceIP string, s
 	vip := util.JoinHostPortInt32(sourceIP, sourcePort)
 	// NOTE: doesn't use vip, to avoid having brackets in the name with IPv6
 	aclName := fmt.Sprintf("%s-%s:%d", lb, sourceIP, sourcePort)
+	aclName = strings.ReplaceAll(aclName, ":", "\\:")
+	// ACL names are limited to 63 characters
+	if len(aclName) > 63 {
```

Comment:
This should be a separate commit, if it's not related to shared-gateway IPv6 support. But also, if this is a problem we're hitting, we should probably just change how we name things, rather than having a complicated truncate-in-some-cases system